code/modules/mob/living/carbon/xenomorph/castes/Runner.dm

```diff
@@ -80,7 +80,7 @@
 
 /mob/living/carbon/xenomorph/runner/recalculate_actions()
 	. = ..()
-	pull_multiplier *= 0.85
+	pull_multiplier *= 0.5
```

Comment:
Don't do this here. Just change the value in the define for `/mob/living/carbon/xenomorph/runner`